package.json

```diff
@@ -95,8 +95,9 @@
     "ember-cli-sass": "^5.6.0",
     "ember-cli-string-utils": "^1.0.0",
     "ember-cli-valid-component-name": "^1.0.0",
+    "ember-fetch": "^1.4.0",
```

Comment:
It looks like active development has moved to `ember-fetch`

https://github.com/tomdale/ember-network/issues/17